.circleci/config.yml

```diff
@@ -385,14 +404,14 @@ jobs:
     steps:
       - checkout
       - restore_cache:
-          key: flowmachine-deps-2-{{ checksum "flowmachine/Pipfile.lock" }}
+          key: flowmachine-deps-3-{{ checksum "flowmachine/Pipfile.lock" }}
       - run:
           name: Linting files with black
-          command: PIPENV_PIPFILE=flowmachine/Pipfile pipenv run black --check .
+          command: pip install black && black --check .
```

Comment:
Won't this install an unpinned version of black?

Reply:
Given this is just for the linter, I didn't think that would be too much of a problem - reckon it's worth pulling the version out of the pipfile?

Reply:
Yeah cos otherwise you'll start randomly getting lint fails when they release a new version.